source/components/GlobalNavigation/components/Notifications/MarkAllButton.js

```diff
@@ -20,9 +20,10 @@ const MarkAllButton = ({ track, header }) => {
             {
                 header && <p className="wds-notifications__header">{header}</p>
             }
-            <button onClick={onClick} className="wds-notifications__mark-all-as-read">
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <a onClick={onClick} className="wds-notifications__mark-all-as-read">
```

Comment:
just curious, why change this from a button to an `a` tag? Are there styles we're re-using or something?

Reply:
exactly. it's defined as an `a` tag in original DS and the styling was done in order to cover that type